.github/workflows/reusable-release.yaml

```
@@ -14,6 +14,13 @@ on:
         required: true
         type: string
         description: "Release tag for all components."
+    secrets:
+      PYPI_API_TOKEN:
+        description: "PyPI API token for publishing Python SDK"
+        required: false
```

```suggestion
        required: true
```

Because `release` job has the `sdk` job as needs